bin/tessel-2.js

```diff
@@ -21,11 +21,16 @@ function makeCommand(commandName) {
     })
     .option('lan', {
       flag: true,
-      help: 'Use LAN connection'
+      help: 'Use only a LAN connection'
     })
     .option('usb', {
       flag: true,
-      help: 'Use USB connection'
+      help: 'Use only a USB connection'
+    })
+    .option('lanPrefer', {
```

Comment:
This is like the `--lan` option but it's not strict. The CLI will use a LAN connection if one is found but will fallback to USB w/o complaining.